docs/topics/about-tools.adoc

```diff
@@ -10,4 +10,5 @@ The {ProductName} ({ProductShortName}) provides several tools to assist you in t
 * link:{ProductDocUserGuideURL}[{CLINameTitle}]
 * link:{ProductDocWebConsoleGuideURL}[{WebNameTitle}]
 * link:{EclipseCrsGuideURL}[{EclipseCrsGuideTitle}]
+* link:{ProductDocVscGuideURL}[Visual Studio Code Extension Guide]
```

Comment:
There is something odd in the rendered doc. #3 and #4 have "Guide in the title while the other docs do not. I think that it should be standardized, so either update the attributes or hard-code the document names in this list.